EIPS/eip-1559.md

```diff
@@ -16,8 +16,8 @@ A transaction pricing mechanism that includes fixed-per-block network fee that i
 There is a base fee per gas in protocol, which can move up or down each block according to a formula which is a function of gas used in parent block and gas target (formerly known as gas limit) of parent block.
 The algorithm results in the base fee per gas increasing when blocks are above the gas target, and decreasing when blocks are below the gas target.
 The base fee per gas is burned.
-Transactions specify both the maximum fee per gas they are willing to pay, as well as the maximum fee per gas they are willing to give to miners to incentivize them to include their transaction (aka: gas premium).
-The transaction will always pay the base fee per gas of the block it was included in, and they will pay the premium per gas set in the transaction, as long as the combined amount of the two fees doesn't exceed the transaction's maximum fee per gas.
+Transactions specify both the maximum network fee per gas (aka: base fee) they are willing to pay, as well as the maximum fee per gas they are willing to give to miners to incentivize them to include their transaction (aka: miner bribe).
```

Comment:
`FEECAP` was previously the absolute maximum that the transaction sender is willing to pay. I think setting the overall maximum and the miner bribe maximum (as previously specified) makes more sense, otherwise the user has to add the two together to work out what the overall maximum fee is.

Reply:
There was no intention to change the behavior here (this particular change was meant to be non-normative).  Do you have a suggestion for better wording than what I have that would make that more clear (ideally without going *too* in depth)?

Reply:
Might be easier to specify the other way around - "transactions specify the maximum fee per gas they are willing to give to miners to incentivize them to include their transaction (aka: miner bribe), they also specify a maximum overall fee per gas they are willing to pay, including both the miner bribe and the network fee (aka: base fee)"

Reply:
```suggestion
Transactions specify the maximum fee per gas they are willing to give to miners to incentivize them to include their transaction (aka: miner bribe).
Transactions also specify the maximum fee per gas they are willing to pay total (aka: fee cap), which covers both the miner bribe and the block's network fee per gas (aka: base fee).
```
How does this sound @bluepintail